scss/atoms/typography/_typography.scss

```diff
@@ -12,11 +12,11 @@
 body {
 	font: $normal #{ map-get( $defaultFont, font-size ) }/1.5 $sans;
 
-	@media ( max-width: 800px ) {
+	@media ( min-width: $viewport-xs ) {
```

Comment:
Just mentioning this again though from way back. I still think its a bad idea to adjust the default font size in a media query. I think this is too heavy handed to apply in general. Seems like something that should be project specific if its going to be done.